src/components/UserProfile/BasicInformationTab/BasicInformationTab.css

```diff
@@ -1,10 +1,9 @@
 .basic-info-tab-tablet {
   display: none;
 }
-@media (max-width: 1199px) {
-  .phone-input-style.phone-input-style {
-    width: 265px !important;
-  }
+.phone-input-style.phone-input-style {
+  width: 100%; 
+  max-width: 300px; 
 }
 
 @media (max-width: 1024px) {
```

Comment:
you could specify that it'll apply to the screens, `@media screen and (max-width: 1024px) {`